app/assets/stylesheets/base/_tables.scss

```diff
@@ -1,32 +1,26 @@
 table {
-  display: flex;
-  flex-flow: column nowrap;
-  font-size: .8em;
-  margin: 0.5em;
-  line-height: 1.5;
-  flex: 1 1 auto;
+  border-collapse: collapse;
+  margin: 20px auto;
+  width: 85%;
 }
 
-th {
-  font-weight: 700;
+td, th {
+  padding-bottom: 5px;
+  border: 5px;
+  padding-left: 5px;
+  padding-right: 5px;
+  text-align: center;
 }
 
-th > td {
-  align-items: center;
-  justify-content: center;
-  white-space: normal;
+th {
+  word-wrap: break-word;
+  border-bottom: $border;
```

Comment:
Can this rule be moved to after line 9? because it is used in both td and th

Reply:
👌